source/Pages/software_on_spider.rst

```diff
@@ -212,6 +213,251 @@ Extraction and Redistribution)` nodes, your Softdrive files will be available un
+
+.. code-block:: bash
+
+	export EB_PYTHON=python3.9
+
+Later if you use EasyBuild installing a software, EasyBuild will generate a module which you can load. To update the module search path environment variable ``$MODULEPATH``, run command:
+
+.. code-block:: bash
+
+	module use /project/<project-name>/Software/easybuild/modules/all
+
+If you may use EasyBuild to install Python packages, you also need to update the Python search path environment variable ``$PYTHONPATH`` to instruct Python where it can find the EasyBuild Python packages:
+
+.. code-block:: bash
+
+	export PYTHONPATH=/project/<project-name>/Software/lib/python3.9/site-packages:$PYTHONPATH
+
+IMPORTANT: keep in mind that you will have to make these environment changes again if you start a new shell session in Spider. To avoid this, you can update one of the shell startup scripts in your home directory. For example you can edit the ``.bashrc`` file found in a users' home directory:
+
+.. code-block:: bash
+
+	cd /home/<user-name>
+	nano /home/<user-name>/.bashrc
+
+Next, add the environment changes to the ``.bashrc`` file and save. 
+
+.. code-block:: bash
+
+	export PATH=/project/<project-name>/Software/bin:$PATH
+	export EB_PYTHON=python3.9
+	export PYTHONPATH=/project/<project-name>/Software/lib/python3.9/site-packages:$PYTHONPATH
+	module use /project/<project-name>/Software/easybuild/modules/all
+
+Now you are good to explore the EasyBuild world!
+
+
+.. _configure-easybuild:
+
+Configure EasyBuild
+=========================
+
+After EasyBuild is installed, here we give instruction on how to configure EasyBuild in Spider. 
+
+To get an overview of the current EasyBuild configuration, run command:
+
+.. code-block:: bash
+
+	eb --show-config
+
+It then shows a couple of selected important configuration settings with default values: build path, install path, path to easyconfigs repository, the robot search path, source path. 
+
+Configure EasyBuild via the environemt variable ``$EASYBUILD_PREFIX`` which changes all inportant configueration settings. To significantly speed up the builds, you can also change the build path to ``/tmp``. Note that the build directories are emptied and removed by EasyBuild when the installation is completed (by default).
+
+.. code-block:: bash
+
+	export EASYBUILD_PREFIX='/project/<project-name>/Software/easybuild'
+	export EASYBUILD_BUILDPATH='/tmp'
+
+For a full description of what you can configure in EasyBuild, please check `Demo on configuing EasyBuild`_.
+
+
+.. _install-software:
+
+Install software
+=========================
+
+Here the most important range of topics are covered if you are new to EasyBuild. ``eb`` is EasyBuild’s main command line tool, to interact with the EasyBuild framework and hereby the most common command line options are being documented.To instruct EasyBuild which software packages it should build/install and which build parameters it should use, one or more easyconfig files must be specified. 
```

Comment:
Mention here that it may take long to install and using a WN is better, and add in line 376 also a warning that a WN is better to install